extensions/wikia/InstantGlobals/InstantGlobalsModule.class.php

```diff
@@ -46,15 +47,16 @@ private function getVariablesValues() {
 	}
 
 	public function getScript( ResourceLoaderContext $context = null ) {
-		global $wgMemc;
 		$variables = null;
-		$cacheKey = wfMemcKey( 'instantGlobalsVariablesValues' );
+		$cacheKey = wfSharedMemcKey( 'instantGlobalsVariablesValues' );
 
-		if ( !( $variables = $wgMemc->get( $cacheKey ) ) ) {
-			$variables = $this->getVariablesValues();
-
-			$wgMemc->set( $cacheKey, $variables, 300 );
-		}
+		$variables = WikiaDataAccess::cache(
+			$cacheKey,
+			self::IG_CACHE_TIME,
```

Comment:
You can use `WikiaDataAccess::DEFAULT_TTL`

Reply:
Ok, changed.